bedrock/firefox/templates/firefox/mobile/index.html

```diff
@@ -59,6 +53,23 @@ <h3>{{ ftl('firefox-mobile-get-automatic-privacy-on-mobile') }}</h3>
             </div>
           </div>
         </div>
+
+        <div class="c-inline-form">
+          <h3>{{ _('Get Firefox for mobile') }}</h3>
+          {% if show_send_to_device %}
+            <p>{{ _('Send a download link to your phone. ') }}</p>
+            {{ send_to_device(include_title=False, message_set='fx-mobile-download-desktop', class='vertical') }}
+          {% else %}
+            <p>{{ _('Scan the QR code to get started') }}</p>
+            <div class="qr-code-wrapper">
+              <img src="{{ static('img/firefox/mobile/protocol/qr-firefox.png') }}"
+                  id="firefox-qr"
+                  data-mozillaonline-link="{{ static('img/firefox/mobile/protocol/qr-firefox-mozillaonline.png') }}"
+                  alt="QR code to scan for Firefox Focus.">
```

Comment:
I left this in because it was in the original page, but this QR code does not seem to be for Firefox Focus (which I think is being sunsetted?)

Reply:
This looks like copy pasta from another page. Suggest adding a new Fluent string for this saying "Scan the QR code to get Firefox on mobile"?